scripts/custom_lint_checks.py

```diff
@@ -623,6 +618,58 @@ def _add_raise_message(self, missing_excs, node):
             node=node)
 
 
+class ImportOnlyModulesChecker(checkers.BaseChecker):
+    """Checker for import from statements. It checks whether
+    modules are imported only.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+
+    name = 'import-only-modules'
+    priority = -1
+    msgs = {
+        'C0003': (
+            'Import \"%s\" from \"%s\" is not a module.',
+            'import-only-modules',
+            'Only modules should be imported.',
+        ),
+    }
+
+    @checker_utils.check_messages('import-only-modules')
+    def visit_importfrom(self, node):
+        """Visits all import from statements in a python file and checks whether
```

Comment:
`...and checks that modules are imported.`

Reply:
Done.